File: Puntonet/Puntonet.Web/Modules/ServerTypes/Parameters/PersonAccesoriesForm.ts
import { IntegerEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface PersonAccesoriesForm {
    IdPerson: IntegerEditor;
    IdAccesory: IntegerEditor;
}

export class PersonAccesoriesForm extends PrefixedContext {
    static formKey = 'Parameters.PersonAccesories';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!PersonAccesoriesForm.init)  {
            PersonAccesoriesForm.init = true;

            var w0 = IntegerEditor;

            initFormType(PersonAccesoriesForm, [
                'IdPerson', w0,
                'IdAccesory', w0
            ]);
        }
    }
}

File: Puntonet/Puntonet.Web/Modules/Parameters/PersonAccesories/PersonAccesoriesForm.cs
using Serenity.ComponentModel;
using Serenity.Web;

namespace Puntonet.Parameters.Forms
{
    [FormScript("Parameters.PersonAccesories")]
    [BasedOnRow(typeof(PersonAccesoriesRow), CheckNames = true)]
    public class PersonAccesoriesForm
    {
        public int IdPerson { get; set; }
        public int IdAccesory { get; set; }
    }
}